inference-engine/tests_deprecated/functional/vpu/myriad_tests/myriad_multiple_graph_tests.cpp

```diff
@@ -27,5 +27,8 @@ TEST_P(myriadMultipleGraphsTests_nightly, LoadGraphsOnDevice) {
 }
 
 INSTANTIATE_TEST_CASE_P(numerOfGraphs, myriadMultipleGraphsTests_nightly,
-    ::testing::Values(2, 4, 10)
+//  Exclude the test cases failing due to known bug:
+//  #-37268: [IE][Myriad] assertion "0 == semInit(&semUsed, 0)" and "stack smashing"
+//  ::testing::Values(2, 4, 10)
```

Comment:
Actually, I think it's better to use common approach to disable tests. It seems to me convenient to avoid disabling statements spreaded over all testing code base.

You can take look at

https://github.com/gladilov-gleb/openvino/blob/master/inference-engine/tests_deprecated/functional/vpu/myriad_tests/vpu_tests_config.cpp

and

https://github.com/gladilov-gleb/openvino/blob/master/inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

Reply:
Does this new way for skipping some tests work for the old-style (deprecated) test suite?

Reply:
Sorry, it looks like it works with new infra only. Resolving.